echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/artifacts/BuildArtifactExtractor.java

```diff
@@ -43,13 +40,13 @@
  */
 @Component
 @Slf4j
-public class ArtifactPostProcessor implements PipelinePostProcessor {
+public class BuildArtifactExtractor {
```

Comment:
So this is specifically for parsing artifacts out of property files? The naming of "BuildArtifactExtractor" confuses me a little. An updated description of the class might be nice.

Reply:
Great point, I renamed it `JinjaArtifactExtractor` and updated the description.  Right now it only knows how to look in properties for the name of the template (as that's the only place there can be) but I can definitely imagine in the future splitting this out into "where to look for the template name" and "extract the artifacts" if more places become supported.

I also realized the description still said it was an unimplemented no-op...which I should probably have updated months ago